pygeos/linear.py

```diff
@@ -140,3 +146,31 @@ def shared_paths(a, b, **kwargs):
     <pygeos.Geometry GEOMETRYCOLLECTION (MULTILINESTRING EMPTY, MULTILINESTRING ...>
     """
     return lib.shared_paths(a, b, **kwargs)
+
+
+def shortest_line(a, b, **kwargs):
+    """
+    Returns the shortest line between geom1 and geom2.
+
+    The resulting line consists of two points, representing the nearest
+    points between the geometry pair. The line always starts in geom1 and
+    ends in geom2. The endpoints of the line will not necessarily be existing
+    vertices of geom1 and geom2, but can be any point along a line segment
```

Comment:
```suggestion
    vertices of geom1 and geom2, but can also be a point along a line segment.
```

The latest remark "(if the input geometries are lines or polygons)." is a bit confusing, I think it doesn't add anything and should be removed